CHANGES.md

```diff
@@ -1,5 +1,88 @@
 # Changelog
 
+## 5.5 (2022-01-26)
+[View detailed changelog](https://github.com/spinalcordtoolbox/spinalcordtoolbox/compare/5.4...5.5)
+
+**FEATURE**
+
+ - **sct_deepseg:** Add models for MP2RAGE SC and MS lesion segmentation.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3637)
+ - **sct_testing:** Bring back `sct_testing` as a light wrapper for `pytest`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3605)
```

Comment:
The release guidelines say to avoid/remove redundancy with the labels like `sct_testing` appearing twice here.